web/src/main/java/org/cbioportal/genome_nexus/web/IsoformOverrideController.java

```diff
@@ -1,87 +0,0 @@
-package org.cbioportal.genome_nexus.web;
```

Comment:
at one point we must have used this endpoint in the msk importer code. There is a property listed in importer.properties.EXAMPLE: "genomenexus.isoform_override="
However, this property is no longer in use in the importer code base.
Similarly, there is a property in genome-nexus-annotation-pipeline: annotationPipeline/src/main/resources/application-test.properties.EXAMPLE  "genomenexus.isoform_override=<HOST>/variant_annotation/isoform_override/"
but there also seems to be no references to this property in the code base any longer.
We should strip unreferenced properties in these other repositories.

Reply:
Right, we should strip properties that are no no longer in use.

Not sure if we ever used this endpoint directly though. It is possible that we've been only using the `isoformOverrideSource` option.